draft-rpc-rfc7322bis.xml

```diff
@@ -660,7 +660,7 @@ the RFC and publish both documents simultaneously.</t>
              <eref target="https://www.rfc-editor.org/info/rfc6429">&gt;https://www.rfc-editor.org/info/rfc6429 &lt;</eref>.</t>
           </section>
           <section anchor="referencing-stds-and-bcps" toc="default">
-            <name>Referencing RFC(s) in a Subseries (STDs, BCPs, and FYIs</name>
+            <name>Referencing RFC(s) in a Sub-series (STDs, BCPs, and FYIs</name>
```

Comment:
Per discussion with the RPC team, "Sub-series" should be "Subseries"

Reply:
updated.